client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -126,18 +131,13 @@ class EditorRevisionsList extends PureComponent {
 				<EditorRevisionsListHeader numRevisions={ revisions.length } />
 				<div className="editor-revisions-list__scroller">
 					<ul className="editor-revisions-list__list">
-						{ isEmpty( revisions ) &&
-							isEmpty( newRevisionIds ) && (
-								<div className={ 'editor-revisions-list__revision-placeholder' } />
+						{ isEmpty( revisions ) && (
```

Comment:
These two can be combined as: `isEmpty( revisions ) || isRequestingRevisions && (…)`